lib/tmx.lua

```diff
@@ -309,24 +309,6 @@ local get_map_tables = function(map_directory, map_file_ext)
   return map_tables
 end
 
-local draw_tiles = function(layer, map)
```

Comment:
Don't forget to remove from this file where you're still returning `draw_tiles`.